pom.client.xml

```diff
@@ -718,14 +718,19 @@
             <version>${maven-surefire-plugin.version}</version>
             <configuration>
               <argLine>
-                <!-- KeyVault tests fail without these exports -->
+                <!-- Mockito uses reflection to create PollResponses in tests. -->
+                --add-opens com.azure.core/com.azure.core.util.polling=ALL-UNNAMED
```

Comment:
nit: Keep all `opens` args together. There's a bunch of `opens` after `exports`.

Reply:
Fixed